### ISSUE TITLE
PDF-X(3)

### ISSUE THREAD

**Issue by tpapp** [JuliaLang/julia-logo-graphics#9]
I am printing a new batch of logos, and while the printer is OK with RGB in SVG, I was told that I would get the most color fidelity in PDF/X (preferably PDF/X-3) using CYMK.

I wonder if we can generate such a file for the logo, either as an asset for this repo or in a release tarball. The intricaties of color management are outside my area of expertise, but if there are tools in Linux to do this I am happy to make a Docker image that automates this.

**Comment by cormullion** [comment]
I can add a PDFX3 CYMK version. A quick comparison on screen suggests that the blue is (as usual) the main victim of the CYMK conversion:

<img width="1147" alt="Screenshot 2020-05-28 at 10 42 08" src="https://user-images.githubusercontent.com/52289/83125875-feb26500-a0cf-11ea-8455-68714555b4bc.png">

**Comment by tpapp** [comment]
Sure, but I would rather have us convert the blue in a standardized way than leaving it to the printer.

**Comment by exaexa** [comment]
Hi everyone, +1ing this.

Just specifying the actual colorspace for the RGB colors (is it adobe RGB?) would help a lot already. Additionally, the usual CMYK conversion tools suck, so having a method in place to convert it with given colorimetric intent in mind would be super helpful. (E.g., how do I print the nice julia blue on coated paper?)

Also, I'm not a big fan of pantone, but naming the pantone spot colors for the printing would be probably the easiest to do this reliably now.

**Comment by cormullion** [comment]
Personally, I avoid this stuff as much as possible, and try to avoid using proprietary software like Adobe and Pantone anyway. Last time I got something printed on paper I got the printer guy to do the matching/conversion for me; the professionals know their stuff and don't get paid if they get it wrong... :) 

I'm more than happy for anyone experienced in this area to develop precise color specifications, and I wish them good fortune!

**Comment by exaexa** [comment]
Yeah, certainly not trying to go help adobe in any way, but esp. for communicating precisely with printers this may help a lot.

Anyway, I assume the current colors are sRGB?

**Comment by cormullion** [comment]
🤷

**Comment by exaexa** [comment]
Anyway, for anyone searching for CMYK values, these are extracted from the pdfx3 version and print much better than srgb:
```
red: 0.0392151 0.875 0.832031 0.0627441
blue 0.765625 0.533203 0 0
violet 0.498047 0.662109 0 0
green 0.673828 0.0392151 0.984375 0.121582
```

**Comment by StefanKarpinski** [comment]
Anyone want to make a PR to add more printer helpful versions to the repo? I think the stance "for communicating precisely with printers this may help a lot" is the right one. We don't dictate what printing technology people may have to use, we just have to work with it and try to make the best of it.

**Comment by exaexa** [comment]
To be future-safe I think one pretty good way would be to agree on CIELAB or CIEXYZ colorspace values, and point people to tools like `icclu` (see http://www.argyllcms.com/ ) that can translate these colorspaces to pretty much any sensible color system out there.
